### PR TITLE
Add missing Date range calculation

### PR DESCRIPTION
**Pull request by shial4**
Apologies for another PR, however, when switching to mainstream, I've noticed I have not committed this to the previous PR.

Skip Pull Request Checklist:

- [x] REQUIRED: I have signed the [Contributor Agreement](https://github.com/skiptools/clabot-config)
- [x] REQUIRED: I have tested my change locally with `swift test`
- [x] OPTIONAL: I have tested my change on an iOS simulator or device
- [x] OPTIONAL: I have tested my change on an Android emulator or device

### CODE REVIEW

File: Tests/SkipFoundationTests/DateTime/TestCalendar.swift [hunk 1]
```diff
@@ -576,6 +576,26 @@ class TestCalendar: XCTestCase {
...
+        let date = Date(timeIntervalSince1970: 1539146705)
+
+        let monthRangeInAYear = calendar.range(of: .month, in: .year, for: date)
+        XCTAssertEqual(monthRangeInAYear, 1..<13)
+        
+        let dayRangeInAYear = calendar.range(of: .day, in: .year, for: date)
+        XCTAssertEqual(dayRangeInAYear, 1..<366)
+        
+        let weekOfYearRangeInAYear = calendar.range(of: .weekOfYear, in: .year, for: date)
+        XCTAssertEqual(weekOfYearRangeInAYear, 1..<54)
```
Comment (marcprux) [inline Tests/SkipFoundationTests/DateTime/TestCalendar.swift:590]: Curious … this is [failing](https://github.com/skiptools/skip-foundation/pull/28/checks#step:9:28221) on the Java side of the local tests: `java.lang.AssertionError: expected:<1..53> but was:<1..52>`

**Review by marcprux — CHANGES_REQUESTED** [review]

File: Tests/SkipFoundationTests/DateTime/TestCalendar.swift (hunk 1 above)
Comment (shial4) [reply Tests/SkipFoundationTests/DateTime/TestCalendar.swift:590]: unfortunately I have no idea what has happen to my env. but I don't think my Java tests are working at all.
getting this on every file kinda thing. 
>Annotation argument must be a compile-time constant.
>Unresolved reference 'ext'.

Comment (marcprux) [reply Tests/SkipFoundationTests/DateTime/TestCalendar.swift:590]: That was a recent fix in the skip-unit framework. Can you run `Update to latest package versions` and do a clean build and let me know if the error still happens?

Comment (shial4) [reply Tests/SkipFoundationTests/DateTime/TestCalendar.swift]: yea, got it working. Did some testing and it seems Swift for weeks in a year always returns max which is not an actual max with in given date (year)

**Review by marcprux — APPROVED** [review]

**Pull request merged**